contracts/Vault.sol

```diff
@@ -169,7 +169,7 @@ contract Vault is
 
     /// See {IVault}
     function yieldFor(address _to)
-        external
+        public
```

Comment:
Is this change intentional?

Reply:
I think it came from reverting the `BaseVault` commit
it was previously external, and delegated to `_yieldFor`, which doesn't exist here